dali/kernels/CMakeLists.txt

```diff
@@ -33,6 +33,21 @@ collect_headers(DALI_INST_HDRS PARENT_SCOPE)
 collect_sources(DALI_KERNEL_SRCS)
 collect_test_sources(DALI_KERNEL_TEST_SRCS)
 
+if (NOT CUSTOM_KERNEL_SRCS_PATTERN STREQUAL "")
+  message("Filtering CUSTOM_KERNEL_SRCS_PATTERN ${CUSTOM_KERNEL_SRCS_PATTERN}")
+  file(GLOB_RECURSE DALI_KERNEL_SRCS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
+       "${CUSTOM_KERNEL_SRCS_PATTERN}")
+  list(APPEND DALI_KERNEL_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/common/scatter_gather.cu")
```

Comment:
What is the reason to append one particular file here?

Reply:
used in libdali. Without it, we can't have a meaningful build

Reply:
I think it's OK to add it by default, but it should be possible to exclude it.